src/app/components/Header/HeaderNavigation.tsx

```diff
@@ -35,11 +36,14 @@ export const HeaderNavigation = ({
 
   return (
     <nav>
-      <ul className="desktop:flex-row desktop:gap-[32px] flex flex-col gap-[12px]">
+      <ul className="flex flex-col gap-[12px] desktop:flex-row desktop:gap-[32px]">
         {navigationHeaderData.map((item, index) => (
           <li
             key={index}
-            className={`text-14 desktop:text-16 group font-medium leading-[1.5] ${index === 1 ? "desktop:relative" : ""}`}
+            className={cn(
+              "group text-14 font-medium leading-[1.5] desktop:text-16",
+              { "desktop:relative": index === 1 },
```

Comment:
It can also be simplified: `index === 1 && "desktop:relative"` without curly brackets around it.